ProcessMaker/processmaker
# FOUR-8704

## Issue & Reproduction Steps
The ScriptExecutorCreated does not have details

## Solution
Avoiding null values

## How to Test
1. Create a Script Executor
2. Go to Admin → Users
3. Check Security Logs

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-8704

## Code Review Checklist

ci:deploy
